lang_js/parsing/parser_js.mly

```diff
@@ -624,6 +624,8 @@ class_element:
     }
 
  | semicolon                       { C_extrasemicolon $1 }
+  /*(* sgrep-ext: enable class body matching *)*/
```

Comment:
excellent. Does this introduce any additional shift/reduce conflict (a warning ocamlyacc displays when
you add an ambiguity in the grammar).
Before this diff there was 2 shift/reduce conflicts (I should fix them, but sometimes it's hard).
Does this diff introduce additional one?

Reply:
I just tested and it does not, but it's nice to say if it does in the Testing plan.

Reply:
@aryx I dont know what "shift/reduce conflict" would be. Any pointer would be appreciated